src/services/etherpad/hooks/Session.js

```diff
@@ -0,0 +1,112 @@
+const { authenticate } = require('@feathersjs/authentication').hooks;
+const { disallow } = require('feathers-hooks-common');
+const { Forbidden } = require('@feathersjs/errors');
+const EtherpadClient = require('../utils/EtherpadClient');
+
+const logger = require('../../../logger');
+
+const getAuthorData = async (context) => {
+	const authorService = context.app.service('etherpad/authors').create({ userId: context.params.account.userId });
+	try {
+		const response = JSON.parse(await authorService);
+		context.data = {
+			...context.data,
+			authorID: response.data.authorID,
+		};
+		return context;
+	} catch (err) {
+		logger.error('Failed to get author data: ', err);
+		throw new Forbidden('Failed to get author data');
+	}
+};
+
+const getGroupData = async (context) => {
+	const groupService = context.app.service('etherpad/groups').create(context.data);
+	try {
+		const response = JSON.parse(await groupService);
+		context.data = {
+			...context.data,
+			groupID: response.data.groupID,
+		};
+		return context;
+	} catch (err) {
+		logger.error('Failed to get course data: ', err);
+		throw new Forbidden('Failed to get course data');
+	}
+};
+
+const getSessionInformation = async (context) => {
+	const sessionListPromise = EtherpadClient.getActiveSessions({ authorID: context.data.authorID });
+	try {
+		const response = JSON.parse(await sessionListPromise);
+
+		// Return existing session from hooks
+		if (typeof (response.data) !== 'undefined' && response.data !== null) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must not be padded by blank lines.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5525245)